views/layout.inc.menu.dt

```diff
@@ -21,8 +21,8 @@
 			ul
 				- foreach(items; listitems)
 					- if(items[1].length < 2)
-						- bool active = startsWith(req.path[1..$], items[1][0]);
-						- if( items[1][0].empty ) active = req.path == "/";
+						- bool active = startsWith(req.requestPath.toString()[1..$], items[1][0]);
```

Comment:
this is unrelated to the change above, please only do one thing at a time in a PR, you can have multiple PRs open at the same time